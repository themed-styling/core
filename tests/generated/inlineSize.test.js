import 'jest-styled-components'

import { inlineSize } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(inlineSize, 'inlineSize')
