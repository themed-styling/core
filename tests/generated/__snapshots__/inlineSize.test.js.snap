// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`inlineSize renders 0 1`] = `
Array [
  .c0 {
  inline-size: 0;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: 0;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: 0;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`inlineSize renders function 1`] = `
Array [
  .c0 {
  inline-size: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: has;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: has;
}

<div
    className="c0"
  />,
]
`;

exports[`inlineSize renders invalid theme path 1`] = `
Array [
  .c0 {
  inline-size: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`inlineSize renders negative number 1`] = `
Array [
  .c0 {
  inline-size: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`inlineSize renders object [breakpoints + standard] 1`] = `
Array [
  .c0 {
  inline-size: July;
}

@media screen and (min-width:November) {
  .c0 {
    inline-size: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    inline-size: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    inline-size: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    inline-size: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  inline-size: July;
}

@media screen and (min-width:November) {
  .c0 {
    inline-size: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    inline-size: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    inline-size: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    inline-size: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  inline-size: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    inline-size: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    inline-size: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    inline-size: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    inline-size: June;
  }
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    inline-size: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    inline-size: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    inline-size: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    inline-size: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`inlineSize renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    inline-size: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    inline-size: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    inline-size: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    inline-size: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    inline-size: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    inline-size: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    inline-size: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    inline-size: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    inline-size: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    inline-size: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    inline-size: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    inline-size: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    inline-size: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    inline-size: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    inline-size: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    inline-size: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`inlineSize renders object [mediaQueries + standart] 1`] = `
Array [
  .c0 {
  inline-size: July;
}

@media May {
  .c0 {
    inline-size: August;
  }
}

@media April {
  .c0 {
    inline-size: September;
  }
}

@media January {
  .c0 {
    inline-size: November;
  }
}

@media April {
  .c0 {
    inline-size: August;
  }
}

@media June {
  .c0 {
    inline-size: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  inline-size: July;
}

@media May {
  .c0 {
    inline-size: August;
  }
}

@media April {
  .c0 {
    inline-size: September;
  }
}

@media January {
  .c0 {
    inline-size: November;
  }
}

@media April {
  .c0 {
    inline-size: August;
  }
}

@media June {
  .c0 {
    inline-size: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  inline-size: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    inline-size: November;
  }
}

@media April {
  .c0 {
    inline-size: January;
  }
}

@media January {
  .c0 {
    inline-size: December;
  }
}

@media April {
  .c0 {
    inline-size: April;
  }
}

@media June {
  .c0 {
    inline-size: May;
  }
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    inline-size: November;
  }
}

@media April {
  .c0 {
    inline-size: January;
  }
}

@media January {
  .c0 {
    inline-size: December;
  }
}

@media April {
  .c0 {
    inline-size: April;
  }
}

@media June {
  .c0 {
    inline-size: May;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`inlineSize renders object [mediaQueries] 1`] = `
Array [
  @media May {
  .c0 {
    inline-size: July;
  }
}

@media April {
  .c0 {
    inline-size: August;
  }
}

@media January {
  .c0 {
    inline-size: September;
  }
}

@media April {
  .c0 {
    inline-size: November;
  }
}

@media June {
  .c0 {
    inline-size: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    inline-size: July;
  }
}

@media April {
  .c0 {
    inline-size: August;
  }
}

@media January {
  .c0 {
    inline-size: September;
  }
}

@media April {
  .c0 {
    inline-size: November;
  }
}

@media June {
  .c0 {
    inline-size: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    inline-size: January;
  }
}

@media April {
  .c0 {
    inline-size: December;
  }
}

@media January {
  .c0 {
    inline-size: April;
  }
}

@media April {
  .c0 {
    inline-size: May;
  }
}

@media June {
  .c0 {
    inline-size: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    inline-size: January;
  }
}

@media April {
  .c0 {
    inline-size: December;
  }
}

@media January {
  .c0 {
    inline-size: April;
  }
}

@media April {
  .c0 {
    inline-size: May;
  }
}

@media June {
  .c0 {
    inline-size: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`inlineSize renders postitive number 1`] = `
Array [
  .c0 {
  inline-size: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`inlineSize renders string 1`] = `
Array [
  .c0 {
  inline-size: July;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: July;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: June;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: June;
}

<div
    className="c0"
  />,
]
`;

exports[`inlineSize renders undefined 1`] = `
Array [
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
]
`;

exports[`inlineSize renders valid theme path 1`] = `
Array [
  .c0 {
  inline-size: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  inline-size: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
